apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -1470,6 +1520,27 @@ def create_network_postcommit(self, context):
             self.apic_manager.update_name_alias(
                 self.apic_manager.apic.fvAEPg, tenant_id, app_profile_name,
                 epg_name, nameAlias=network_name)
+            # set the tenant or application profile nameAlias if this is the
+            # first network being created under this tenant
```

Comment:
In Neutron it might be irrelevant whether or not you create the alias for a tenant (as you said, networks are literally the first kind of objects you can create that affect ACI). But let's have more discussion for the GBP case.